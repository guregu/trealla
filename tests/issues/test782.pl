:- initialization(main).

main :-
	L = ""||K,
	write(L=K), nl.
