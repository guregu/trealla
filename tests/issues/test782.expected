_4=_4
